### DISCUSSION TITLE
magento 2 extension

### DISCUSSION THREAD

**Discussion by Asifjalil99** [chitchats/chitchats-api-doc#40] [Q&A]
is there any magento 2 extension available for ChitChats ??/

**Comment by rmarescu** [comment]
Hey @Asifjalil99, thanks for reaching out. At this moment we do not have an integration for Magento. We had some discussions internally about what platforms should be considered for future integrations and Magento was on the list, although not at the top.

You can find all supported platforms on the [integrations page](http://chitchats.com/integrations). The page will be updated in the future when new integrations are completed.
